pkg/kopia/repository/create.go

```diff
@@ -0,0 +1,152 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/kanisterio/kanister/pkg/format"
+	"github.com/kanisterio/kanister/pkg/kopia/command"
+	kerrors "github.com/kanisterio/kanister/pkg/kopia/errors"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+	"github.com/kanisterio/kanister/pkg/utils"
+)
+
+const (
+	// Kopia returned errors
+	errCodeOutOfMemory = "command terminated with exit code 137"
+	errAccessDenied    = "Access Denied"
+	errInvalidPassword = "invalid repository password"
+)
```

Comment:
These error messages are already present in `pkg/kopia/errors/errors.go`
Can we make use of those?